velox/parse/Expressions.h

```diff
@@ -205,63 +141,6 @@ class FieldAccessExpr : public core::IExpr {
   VELOX_DEFINE_CLASS_NAME(FieldAccessExpr)
 };
 
-class SortExpr : public core::IExpr {
```

Comment:
out of curiosity: what's the "correct" way to represent sorting from a parsed SQL statement?

Reply:
There is a special API for that: parseOrderByExpr

```
// Parses an ORDER BY clause using DuckDB's internal postgresql-based parser,
// converting it to a pair of an IExpr tree and a core::SortOrder. Uses ASC
// NULLS LAST as the default sort order.
std::pair<std::shared_ptr<const core::IExpr>, core::SortOrder> parseOrderByExpr(
    const std::string& exprString);
```